src/golang/lib/models/shared/storage.go

```diff
@@ -44,6 +49,10 @@ type S3Config struct {
 type S3ConfigPublic struct {
 	Region string `yaml:"region" json:"region"`
 	Bucket string `yaml:"bucket" json:"bucket"`
+
+	// Use this directory in the bucket as the root. If not set, we default to the root of the bucket.
+	// Expected to be santizied into the format "path/to/dir/" (without a leading slash, but with a trailing one).
+	RootPath string `yaml:"root_dir" json:"root_dir"`
```

Comment:
Is there a reason we're using different variable name here? It's `RootDir` above.

Reply:
nope, I went back and forth on this name so it probably was just left behind.